frontend/src/components/Redirect.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, SyntheticEvent, useEffect } from "react";
+import styles from "../styles/Buttons.module.css"
+import axios from "axios";
+import { routes } from "../util/config";
+import { useParams, useNavigate, Link } from "react-router-dom";
+
+export const Redirect = (): JSX.Element => {
+  const { id } = useParams()
+  const organizationName = "Brown Hope"
+  const applicationName = "Black Resilience Fund 2021 Application Form"
+  const interviewDate = "June 21"
+  const intakeCallsDate = "July 1"
+  const specialCases = "a food box or mutual aid"
+  // need to find right name for these variables. 
+  const infoURL = "https://www.blackresiliencefund.com/faq"
+  const contactEmail = ""
+  const contactURL = "https://www.brownhope.org/contact"
```

Comment:
The appropriate contact address for this would be info@blackresiliencefund.com

Reply:
Just realized that's an URL and not an email, but yeah Tracy would like this email to be present for questions about the application on this page and the pages that show status.